### PR TITLE
LPS-191322 | Master

### PR DESCRIPTION
**Pull request by kristinonias**
https://liferay.atlassian.net/browse/LPS-191322

cc: @patriciajeanperez

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase [hunk 1]
```diff
@@ -1485,6 +1485,7 @@ definition {
 	@priority = 3
 	test IconOnModalTitleIsGreyWhenItemIsCompletedOnTime {
 		property test.liferay.virtual.instance = "false";
+		property portal.acceptance = "quarantine";
```
Comment (kristinonias) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:1488]: These tests are consistently failing on CI after the test has been fixed due to timer/status.

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Thank you, @kristinonias! Are these added to the spreadsheet?

**Comment by kristinonias** [comment]
Just added them! @patriciajeanperez

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**